other-docs/guides/migrating-from-wordpress.md

```diff
@@ -0,0 +1,150 @@
+# Migrating a WordPress Codebase
+
+This guide covers how to migrate a typical WordPress project to HM Platform.
+
+### Setup Composer
+
+If the project does not already have a `composer.json` in the root of the repository, you should run `composer init` from the project root. This will prompt for any fields. For now, don’t bother adding any dependancies. If you already have a `composer.json` move to Remove Dependencies.
+
+### Remove Dependencies
+
+We’ll need to remove all the dependencies that are part of HM Platform, as those will be installed by HM Platform itself. Firstly, remove the any submodules that are included in HM Platform. For example, `wordpress` and `hm-platform` submodules. To correctly remove a submodule:
+
+1. `git submodule deinit -f wordpress`
+2. `git rm -rf wordpress`
+3. `rm -rf .git/modules/wordpress`
+
+Do the same for `hm-platform` if you have this project dependency. HM Platform also includes the following WordPress plugins, so if you project uses git submodules or uses composer to include them, you should remove them from your project (unless you plan to disable the HM Platform module that makes use of the plugin, and continue with the plugin directly):
+
+- `smart-media`
+- `gaussholder`
+- `aws-rekognition`
+- `two-factor`
+- `stream`
+- `wp-simple-saml`
+- `delegated-oauth`
+- `elasticpress`
+- `hm-redirects`
+- `msm-sitemap`
+- `wp-seo`
+- `amp`
+- `facebook-instant-articles-wp`
+- `meta-tags`
+- `query-monitor`
+- `hm-gtm`
+- `workflows`
+- `wp-user-signups`
+
+### Move wp-config.php
+
+Installing HM Platform will replace your `wp-config.php` so you should back it up if you have application level configuration in your `wp-config.php`.
+
+### Installing HM Platform
+
+Now you have your composer project up and running, it’s time to add HM Platform to the project. Do so by running `composer require humanmade/platform`.
+
+Once HM Platform has been installed, you should see the `wordpress` directory back in the project root, a new `wp-config.php` and `index.php`. You should now also see these 3 paths added to your `.gitignore`.
```

Comment:
They won't be added to an existing one I think?

Reply:
Ah good point, updated